### ISSUE TITLE
Pretty checkpoint loading log

### ISSUE THREAD

**Issue by ZwwWayne** [open-mmlab/mmcv#1113]
For now, the checkpoint loader will return two lines of log:
```
load checkpoint from xxx
Use xxx loader
```
We can simply make it in one line like
```
load checkpoint from {local|ceph|pavi|network} path: xxxxx
```

**Comment by ZwwWayne** [comment]
Please @hhaAndroid take a look

**Comment by zhouzaida** [comment]
closed by #1446
